tests/pauli/test_pauli_utils.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Unit tests for the :mod:`pauli` utility functions in ``pauli/utils.py``.
+Unit tests for the :mod:`pauli` utility functions in ``pauli/.utils.py``.
```

Comment:
```suggestion
Unit tests for the :mod:`pauli` utility functions in ``pauli/utils.py``.
```
Don't know how this popped up in this PR 😅

Reply:
Oops I thought I removed that, it was a silly thing that my IDE refactored.